---
title: "Influence of GAV Transmission in *Penaeus monodon* from Broodstock to Offspring"
author: Cheryl
date: "8 September 2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning=FALSE, results=FALSE, message=FALSE)
```

###Abstract

Gill associated virus (GAV) occurs commonly in black tiger prawn (Penaeus monodon) broodstock in eastern Australia. The virus is believed to be primarily transmitted vertically from broodstock to progeny and is the major route by which infection enters farm ponds to cause disease and production losses. This study investigates the extent to which GAV infection loads in wild broodstock (n=184) are reflected in their progeny reared in commercial ponds at the time of harvest, and the extent to which vertical and/or horizontal transmission, as well as genetic differences between families, might have contributed to infection prevalence and severity between different families. This study looks at 184 broodstock animals and eight ponds of harvest age offspring across three cohorts and demonstrates the impact early-life horizontal transmission, vertical transmission and horizontal transmission have on progeny loads.


###Introduction

In eastern Australia, gill-associated virus (GAV) occurs commonly in black tiger shrimp, Penaeus monodon and is responsible for significant production losses in terms of growth and survival (de la Vega et al., 2004). While demonstrated to be transmitted vertically (Cowley et al., 2002), the implications of farming seedstock derived from wild broodstock differing in GAV infection loads have not been investigated pragmatically. To address this, SNP-based genotyping and reverse transcription real-time quantitative PCR (RT-qPCR) was used to correlate GAV infection prevalence and loads across three cohorts of wild broodstock and their progeny sampled at random at the time of harvest from ponds. 
To identify potential correlations between mean GAV viral loads and prevalence at the family, pond and cohort levels, GAV loads (viral copies/ug TNA) from broodstock and harvest age offspring were measured and compared between contributing parent pairs and corresponding progeny at harvest from three cohorts. Reverse transcriptase real time quantitative PCR (RT-qPCR) testing of total nucleic acid (TNA) extracted from pleopod tissue sampled at eye-stalk ablation was applied to all broodstock. In addition, a subset of broodstock were sampled and viral screened at pre- and post-spawn time points to demonstrate that viral infection load was influenced by stress and handling during these time points.


###Materials and Methods

*Pleopod and gill tissue samples

Tissue samples from wild broodstock, post spawned broodstock and postlarval progeny of these broodstock were obtained from a commercial farm in north-east Queensland. All shrimp tissue samples were collected in 96-deepwell plates and preserved in 2mL of RNA-later solution.

*TNA extraction and GAV RT-qPCR

Total RNA was extracted according to the Kingfisher extraction protocol (Table 1). A 1.5 µL aliquot was examined using a Nanodrop® ND-8000 spectrophotometer (Thermo Fisher) to determine the RNA concentration and relative purity. cDNA was synthesised in a 10 μL reaction containing 500 ng total RNA, 0.5 μL reverse transcriptase and 2 μL 5x TransAmpBuffer (Bioline Sensifast cDNA Kit). A TaqMan RT-qPCR test for GAV (de la Vega et al., 2004) was performed as described except that 2 µL cDNA (equivalent to 100 ng total RNA) was used in a 20 µL reaction prepared using TaqMan® Universal PCR Master Mix (Applied Biosystems) and 18 µM each PCR primer. From this 20 µL mixture, 3 x 5 µL aliquots were placed into 3 wells of a 384-well PCR plate as plate replicates. QPCR was performed in a Viia7® Real-Time PCR system (Applied Biosystems) using thermal cycling conditions xxxxx . To quantify GAV RNA copy numbers accurately, cDNA prepared to a 10-fold dilution series of synthetic GAV RNA of known copy number was amplified in the same plate to generate a linear regression plot of mean cycle threshold (Ct) value vs RNA copy number. Infection loads were expressed as GAV RNA copies per 1 ng total RNA to adjust for the use of cDNA prepared to 25 ng total RNA in each 5 µL reaction aliquot.

*Genotyping and parentage assignment

The data used in this study were run on two platforms. Cohorts 201602 and 201604 were run on two 60-SNP marker Sequenom® iPLEX Platinum panels for the Black Tiger shrimp (AGRF , Brisbane, Australia) and cohort G1T1701 was run on a DaRTcap platform (Diversity Arrays, ACT, Australia)

*Statistical Analyses

GAV RNA copy number data were adjusted to log10 scale for standardization. Families with five or more individuals per family per pond were selected. For animals tested as “zero”, these were converted to loadings of 0.0001 to avoid them dropping out of the analyses. 


###Results

1.1 Detection of GAV RNA loads pre and post-spawn in broodstock animals
Among the 28 wild broodstock animals sampled at pre-spawn, xx% of them showed a low to moderate low infection level (xx –xx GAV RNA copies/µg TNA).A second cohort (G1T1701_DBR) was tested for GAV infection loads at the same two time points and all animals showed at least a two fold increase in GAV infection levels. 


1.2 GAV RNAcopies/ug TNA across 3 cohorts of broodstock
```{r broodstock, echo = FALSE, warning=FALSE, results=FALSE, message=FALSE}
library(tidyverse)
library(ggplot2)
allbroodstock <- read_csv("https://raw.githubusercontent.com/cheryltky/ViralTransmission/master/Working%20Files/allcohorts_broodstock.csv")
head(allbroodstock)
summary(allbroodstock)
str(allbroodstock)
ggplot(allbroodstock, aes(BRNO, y = Lggav))+
  geom_violin()+
  geom_dotplot(binaxis='y', stackdir='center', dotsize=0.7)+
  theme_classic()

```


In broodstock (n=28) for which a pleopod was also sampled when culled at the end of the hatchery run, they were generally found to have increased.  GAV prevalence and loads of offspring from eight ponds were measured and compared against GAV loads from contributing parents.

```{r all families}
allfams <- read_csv("https://raw.githubusercontent.com/cheryltky/ViralTransmission/master/Working%20Files/allcohortfamsv2.csv")
head(allfams)
summary(allfams)
str(allfams)

allfams <- allfams %>% mutate_if(is.character, as.factor)
allfams$IntFamID <- as.factor(allfams$IntFamID)
allfams$PrawnID <- as.factor(allfams$PrawnID)
allfams$Class <- as.factor(allfams$Class)
allfams$Age <- as.factor(allfams$Age)
allfams$Cohort <- as.factor(allfams$Cohort)
allfams$FamID <- as.factor(allfams$FamID)
str(allfams)
summary(allfams)

```

1.3 GAV RNA copies/ug TNA across all families
```{r scatterplot by families}
ggplot(allfams , aes(IntFamID , GAV)) +
  geom_point() +
  theme_classic()
```

1.4 Distribution of GAV RNA copies/ug TNA across families
```{r jitterplot by families}
allcohortfams <- ggplot(allfams , aes(IntFamID , GAV)) +
  geom_jitter(width = 0.2) +
  aes(color = Age , shape = Class ) +
  labs( title = "All Families GAV Loads",
        x = "Family" , 
        y = "GAV RNA copies/ug TNA")

allcohortfams
ggsave("allcohortfams.png" , allcohortfams , width = 15, height = 10 , dpi = 300)
```


```{r top10famscohort201604}
toptenfams <- read_csv("https://raw.githubusercontent.com/cheryltky/ViralTransmission/master/Working%20Files/201602_04_10fams_26112019.csv")
head(toptenfams)
str(toptenfams)
summary(toptenfams)
toptenfams$IntFamID <- as.factor(toptenfams$IntFamID)
toptenfams$PrawnID <- as.factor(toptenfams$PrawnID)
toptenfams$Class <- as.factor(toptenfams$Class)
toptenfams$Age <- as.factor(toptenfams$Age)
toptenfams$Cohort <- as.factor(toptenfams$Cohort)
toptenfams$FamID <- as.factor(toptenfams$FamID)
str(toptenfams)
summary(toptenfams)

```


```{r cohort1201602topfams}
cohort1topfams <-read_csv("https://raw.githubusercontent.com/cheryltky/ViralTransmission/master/Working%20Files/201602topfams.csv")
head(cohort1topfams)
str(cohort1topfams)
summary(cohort1topfams)

cohort1topfams$PrawnID <- as.factor(cohort1topfams$PrawnID)
cohort1topfams$Class <- as.factor(cohort1topfams$Class)
cohort1topfams$Age <- as.factor(cohort1topfams$Age)
cohort1topfams$Cohort <- as.factor(cohort1topfams$Cohort)
cohort1topfams$FamID <- as.factor(cohort1topfams$FamID)
str(cohort1topfams)
summary(cohort1topfams)

cohort1top10fams <- ggplot(cohort1topfams, aes(IntFamID, GAV)) +
  geom_jitter(width = 0.25) +
  aes(color = Class, shape = Class) +
  labs(title = "Cohort 1 Top GAV Families",
       x = "Family ID",
       y = "Log10GAVcopies/ug TNA") +
  theme_bw()

cohort1top10fams
```



```{r cohort2201604scatterplot}
ggplot(toptenfams , aes(IntFamID, GAV)) +
  geom_point() + 
  theme_classic()

Topfams20160204 <- ggplot(toptenfams , aes(IntFamID, GAV)) +
  geom_jitter(width = 0.25) +
  aes(color = Age, shape = Class) +
  labs(title = "Cohorts 1 and 2 Top 10 families",
       x = "Family ID",
       y = "Log10GAV copies/ug TNA") +
  theme_bw()

Topfams20160204

ggsave("Topfams20160204.png", Topfams20160204 , width = 15, height = 10, dpi = 300)
  

```


```{r 201701fams}
cohort3topfams <- read_csv("https://raw.githubusercontent.com/cheryltky/ViralTransmission/master/Working%20Files/201701top10fams.csv")
head(cohort3topfams)
str(cohort3topfams)
summary(cohort3topfams)

cohort3topfams <- ggplot(cohort3topfams, aes(IntFamID, GAV)) +
  geom_jitter(width = 0.25) +
  aes(color = Age, shape = Class) +
  labs(title = "Cohort 3 Top 10 GAV Families" ,
       x = "Family ID",
       y = "Log10GAVcopies/ug TNA") +
  theme_bw()

cohort3topfams
ggsave("Topcohort3fams.png", cohort3topfams, width = 15, height = 10, dpi = 300)
```

To find out the strength of correlation between viral loads of offspring and the parents, I performed a correlation matrix test

#Discussion